pages/test.tsx

```diff
@@ -1,45 +1,98 @@
-export const Test = () => {
+import { LinkedButton } from "./components/LinkedButton";
+import { useState } from "react";
+
+const Test = ({ setStoppingTime }) => {
```

Comment:
Since you already started doing it, maybe add `type TestProps = ` here

Reply:
Good idea! Done :)